pkgs/development/tools/build-managers/dub/default.nix

```diff
@@ -1,4 +1,6 @@
-{ lib, stdenv, fetchFromGitHub, curl, dmd, libevent, rsync }:
+{ lib, stdenv, fetchFromGitHub, curl, libevent, rsync, ldc ? null, dcompiler ? ldc }:
```

Comment:
```suggestion
{ lib, stdenv, fetchFromGitHub, curl, libevent, rsync, ldc, dcompiler ? ldc }:
```

Reply:
SuperSandro2000  will do, but you are welcome to take it, fix it and merge :-p ;-) :-)
thanks for your effort and patience all, i077 of course that means you in particular too  (hope it is not presumptuous to assume this is done eventually :)

Reply:
Thanks for taking the initiative on this!

It's generally not a good idea to leave it to the reviewer to make the change, they typically do a lot of these :P
Could you just make the change and force-push? Thanks!

Reply:
thought i already had ... think i have already ...
(point taken though) (thanks for the mentoring)